src/clj/rems/form_validation.clj

```diff
@@ -4,16 +4,28 @@
             [rems.common.form :as form]
             [rems.common.util :refer [+email-regex+]]))
 
+(defn- all-columns-set? [field]
+  (let [valid-row? (fn [cells] (every? #(not (str/blank? (:value %))) cells))]
```

Comment:
`not-any?`? I think it reads slightly better here.

Reply:
yeah!